lib/generators/rafinesque/install_generator.rb

```diff
@@ -0,0 +1,16 @@
+require 'rails/generators'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.